reaktive/src/commonMain/kotlin/com/badoo/reaktive/maybe/FlatMap.kt

```diff
@@ -30,4 +30,11 @@ fun <T, R> Maybe<T>.flatMap(mapper: (T) -> Maybe<R>): Maybe<R> =
                 }
             }
         )
-    }
\ No newline at end of file
+    }
+
+fun <T, U, R> Maybe<T>.flatMap(
+    mapper: (T) -> Maybe<U>,
+    resultSelector: (T, U) -> R
+): Maybe<R> = flatMap { t ->
```

Comment:
Please wrap line after `=` in case of multi-line function body. Also consider single for the method signature, it is less than 100 symbols. Same for other cases.

Reply:
I'm not sure if I understood you correctly. Please review changes